middleware/proxy/README.md

```diff
@@ -26,7 +26,11 @@ proxy FROM TO... {
     health_check PATH:PORT [DURATION]
     except IGNORED_NAMES...
     spray
-    protocol [dns [force_tcp]|https_google [bootstrap ADDRESS...]|grpc [insecure|CA-PEM|KEY-PEM CERT-PEM|KEY-PEM CERT-PEM CA-PEM]]
+    protocol [dns [force_tcp]|grpc [insecure|CA-PEM|KEY-PEM CERT-PEM|KEY-PEM CERT-PEM CA-PEM]]
+    protocol https_google [bootstrap ADDRESS...] {
+      padding
+      pinset 0uXWIwuNdaaFSx7xJySymocwKDFJeEVzlouiy6MZATA=
```

Comment:
pinset seems not to be used and even if we need it (I don't know what it is), it should be in another PR.